### PR TITLE
Make FlutterTest the default test font

### PR DESCRIPTION
**Pull request by LongCatIsLooong**
Make FlutterTest the default test font. Fixes https://github.com/flutter/flutter/issues/62819

## Pre-launch Checklist

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [ ] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt. See [testing the engine] for instructions on writing and running engine tests.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I signed the [CLA].
- [ ] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[C++, Objective-C, Java style guides]: https://github.com/flutter/engine/blob/main/CONTRIBUTING.md#style
[testing the engine]: https://github.com/flutter/flutter/wiki/Testing-the-engine
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: lib/web_ui/test/canvaskit/skia_font_collection_test.dart [hunk 1]
```diff
@@ -181,5 +181,14 @@ void testMain() {
...
       expect(warnings, isEmpty);
     });
+
+    test('FlutterTest is the default test font', () async {
+      final SkiaFontCollection fontCollection = SkiaFontCollection();
+
+      await fontCollection.debugDownloadTestFonts();
+      fontCollection.registerDownloadedFonts();
+      expect(fontCollection.debugRegisteredFonts, isNotEmpty);
+      expect(fontCollection.debugRegisteredFonts?.first.family, 'FlutterTest');
```
Comment (mdebbar) [inline lib/web_ui/test/canvaskit/skia_font_collection_test.dart:191]: Since you expect this to always be non-null, maybe use `!` instead of `?` ?

**Review by mdebbar — APPROVED** [review]
Web changes LGTM!

File: lib/web_ui/test/canvaskit/skia_font_collection_test.dart (hunk 1 above)
Comment (LongCatIsLooong) [reply lib/web_ui/test/canvaskit/skia_font_collection_test.dart]: Done.

**Comment by LongCatIsLooong** [comment]
Heads-up: this is going to introduce some (maybe a hundred or so) golden changes in framework tests. I've looked at the ~100 changes on Linux nothing stood out to me. They all should be simple text shifts, there shouldn't be any line-breaking changes. /cc the engine sheriff @flar

**Pull request merged**